packages/styleguide/stories/CodeByteEditor.stories.mdx

```diff
@@ -20,11 +20,30 @@ import { Canvas, Meta, Story } from '@storybook/addon-docs/blocks';
 
 <Canvas>
   <Story
-    name="default"
+    name="Language Provided"
     args={{
       text:
         'console.log("Welcome to the Codebytes Editor!");',
-      language: 'javascript',
+      snippetsBaseUrl: process.env.CONTAINER_API_BASE,
```

Comment:
which ticket did we say we'd handle the baseurl in?

Reply:
I just added a ticket for this. It's one of those tickets that can be done at any point, too.  https://codecademy.atlassian.net/browse/DISC-428